spec/requests/api/v1/auth/token_validations/validate_token_spec.rb

```diff
@@ -0,0 +1,39 @@
+describe 'GET api/v1/auth/token_validations', type: :request do
+  let(:user) { create(:user) }
+  let(:headers) { auth_headers(user) }
+
+  subject do
+    get api_v1_auth_validate_token_path, headers: headers, as: :json
+    response
+  end
+
+  context 'with valid credentials' do
+    it { is_expected.to have_http_status(:ok) }
+
+    it 'returns the logged in user\'s data' do
```

Comment:
I'd rather use `"` and not have to escape the `'`, thoughts?

Reply:
mmm I agree that having to scape when describing tests it at least odd. I would say that if you agree on using " we will have to change .rubocop file to exclude the rspec when analyzing for single quotes and change all the existent tests to use " instead of '. What do u think ? I don't have problems on going that way.

Reply:
rubocop only complains if you use `"` on simple strings. if you say for example `"it's me"` then rubocop doesn't complain

Reply:
⏫